src/openzaak/components/catalogi/signals.py

```diff
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2019 - 2020 Dimpact
+import logging
+
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.db.models.base import ModelBase
+from django.db.models.signals import ModelSignal, post_delete
+from django.dispatch import receiver
+
+from vng_api_common.authorizations.models import Autorisatie
+
+from openzaak.components.catalogi.models.zaaktype import ZaakType
+
+logger = logging.getLogger(__name__)
+
+
+@receiver(post_delete, dispatch_uid="catalogi.sync_autorisaties")
+def sync_autorisaties(
+    sender: ModelBase, signal: ModelSignal, instance: ZaakType, **kwargs
+) -> None:
```

Comment:
Technically `instance` is a `Model` instance, as it will also fire for other models. You can limit the senders though: https://docs.djangoproject.com/en/2.2/topics/signals/#connecting-to-signals-sent-by-specific-senders

Additionally, this applies not only for `ZaakType`, but also for `InformatieObjectType` and `BesluitType`